### ISSUE TITLE
shopping v2 viewport issue

### ISSUE THREAD

**Issue by vabene1111** [TandoorRecipes/recipes#1567]
### Tandoor Version

1.0.8

### Setup

Docker / Docker-Compose

### Reverse Proxy

Traefik

### Bug description

on demo/hosted 

the list is not shown full size at least on some devices. Tested on phone without google only firefox/opera but also on normal recent android with recent chrome version 

no inidication of anything wrong in any of the logs, not sure how to reproduce

https://user-images.githubusercontent.com/6819595/155133671-94f420cd-ca24-40d2-ac23-edee3e7685b4.mp4

**Comment by MaxJa4** [comment]
Same issue on Safari, iPhone.

**Comment by smilerz** [comment]
> interesting, @smilerz you are also using this on iphone, is this happening for you as well ?

No, I haven't seen this problem.  What size device is it happening on?  There are some screen dimension CSS rules that may be the source of the problem.

**Comment by MaxJa4** [comment]
iPhone 12 Pro (not Max) with 6.06" according to Google.

**Comment by smilerz** [comment]
390x844 pixels - I don't have any way to run safari at that viewport size.  @vabene1111 you do development on Mac don't you?

**Comment by vabene1111** [comment]
no, i hate apple i must admit 😂 i dont own a single of their devices

**Comment by smilerz** [comment]
oh weird - I thought I remember there being some vue dev packages exclusively for mac and assumed they came from you.

**Comment by vabene1111** [comment]
ok i just had a similar issue when adding something on mobile due to the height change introduced by the keyboard 

![image](https://user-images.githubusercontent.com/6819595/155493816-9149ddb2-4bfc-47a3-b08d-7c34330b7a2b.png)

**Comment by vabene1111** [comment]
![image](https://user-images.githubusercontent.com/6819595/155875621-3fd0491f-61e1-4f4b-bae0-fdb6d5e917ba.png)
reproducable on chrome repsonsive device viewer, trying to fix now
